api/v1alpha1/webhooks/author/webhook.go

```diff
@@ -148,18 +148,18 @@ func (a *authorWebhook) patchResponse(raw []byte, object client.Object) admissio
 }
 
 // setAuthorLabel returns the passed object with the author label added.
-func (a *authorWebhook) setAuthorLabel(username string, obj client.Object) {
+func (w *Webhook) setAuthorLabel(username string, obj client.Object) {
 	labels := make(map[string]string)
 	if obj.GetLabels() != nil {
 		labels = obj.GetLabels()
 	}
 
-	labels[metadata.AuthorLabel] = a.sanitizeLabelValue(username)
+	labels[metadata.AuthorLabel] = w.sanitizeLabelValue(username)
 	obj.SetLabels(labels)
 }
 
 // sanitizeLabelValue takes a username and returns it in a form appropriate to use as a label value.
-func (a *authorWebhook) sanitizeLabelValue(username string) string {
+func (w *Webhook) sanitizeLabelValue(username string) string {
```

Comment:
How is this change necessary for adding toolkit and not a rewrite?

Reply:
Consistency.